File: All Codes/URI_1035.java
import java.util.Scanner;
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class URI_1035 {
    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            int A = input.nextInt();
            int B = input.nextInt();
            int C = input.nextInt();
            int D = input.nextInt();

            if (B>C && D>A && C+D>A+B && C>0 && D>0 && A%2==0) {
                System.out.printf("Valores aceitos\n");
            } else {
                System.out.printf("Valores nao aceitos\n");
            }
        }
    }
}

File: All Codes/URI_1035.py
a = [int(x) for x in input().split()]
# a[0] = a
# a[1] = b
# a[2] = c
# a[3] = d

con = True if ((a[0] % 2 == 0) and 
               (a[2] > 0) and (a[3] > 0) and 
               ((a[2] + a[3]) > (a[0] + a[1])) and
               (a[3] > a[0]) and
               (a[1] > a[2])) else False

if con:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")